src/interface/AuthInfo/AuthInfo.tsx

```diff
@@ -0,0 +1,35 @@
+export interface AxiosErrorResponse {
+    data: {
+      message: string;
+    };
+    [key: string]: any;
+  }
+  
+  export interface LoginFormData {
+    email: string;
+    password: string;
+  }
+  
+  export interface RegisterFormData extends LoginFormData {
+    userName: string;
+    country: string;
+    phoneNumber: string;
+    confirmPassword: string;
+    profileImage: File | null; // توضيح نوع الصورة بدلاً من استخدام any.
+  }
+  
+  export interface ForgetPasswordFormData {
+    email: string;
+  }
+  
+  export interface ResetPasswordFormData
+    extends LoginFormData,
+      Pick<RegisterFormData, "confirmPassword"> {
```

Comment:
Pick utility's main purpose is to extract complex types from a certain type. It shouldn't be used to extract simple types like "confirmPassword". This will make the code difficult to read for new TypeScript users.
I suggest putting confirmPassword type directly.